api/v1alpha1/apidefinition_types.go

```diff
@@ -45,6 +45,10 @@ type ApiDefinitionStatus struct {
 	// LinkedToAPIs is a list of ApiDefinition namespaced/name that this resource
 	// links to.
 	LinkedToAPIs []model.Target `json:"linked_to_apis,omitempty"`
+
+	// SDL holds introspection results of the subgraph. Therefore, spec.graphql.subgraph.sdl is no longer needed to be
+	// provided manually.
+	SDL string `json:"sdl,omitempty"`
```

Comment:
We don't need this now

Reply:
Shouldn't it stay in the ApiDefinition spec as well? So that our ApiDefinition CRD is compatible with the regular api-definition.

Of course, the suggested way of creating subgraphs must be through SubGraph CRD.

Reply:
It is stored in Status field of APIDefinition. We don't required now with our current approach.